views/company.handlebars

```diff
@@ -1,12 +1,262 @@
 <h1>Stats for {{name}}</h1>
 <div class="companyData">
-    <h3>Zapp Data</h3>
-    Zapp driven hibernations: {{data.zappHibernations}}
+  <h3>Zapp Data</h3>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Metric</th>
+        <th>Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Cumulative Zapp driven hibernations</td>
+        <td>{{dashboardData.cumulativeData.zappHibernations}}</td>
+      </tr>
+      <tr>
+        <td>Cumulative Zapp driven heating actions</td>
+        <td>{{dashboardData.cumulativeData.heatingActions}}</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<div class="chartContainer">
+  <h4>Zapp Users</h4>
+  <canvas id="usersChart"></canvas>
+  <script>
+    var ctx = document.getElementById('usersChart').getContext('2d');

```

Comment:
We should pull these scripts out in to separate files and tidy them up a bit. There's a bit of inconsistency around bracing and spacing, etc. and it looks like some things could be commonised.